SU-HKKU/cardinal_service_profile
# CSD-000 Homepage filter fixup

# READY FOR REVIEW

# Summary
- Fix homepage search filter styles

# Needed By (Date)
- asap

# Urgency
- low

# Steps to Test

1. Pull this PR
2. Clear site cache
3. Go to homepage, select a length Open To or Service theme. Confirm that the search filters do not grow or shrink.
![image](https://user-images.githubusercontent.com/34019925/113357478-ed8e4200-92f8-11eb-896f-694e2cd3a9ea.png)

It should appear like this:
![image](https://user-images.githubusercontent.com/34019925/113357528-fda62180-92f8-11eb-88b0-da0917211f7d.png)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
